docusaurus/docs/reference/ui-extensions/_category_.json

```
@@ -0,0 +1,6 @@
+{
+  "position": 50,
+  "label": "UI Extensions",
```

```suggestion
  "label": "UI extensions",
```
(assuming we don't remove UI yet_